tests/glsl-intrinsic/shader-subgroup/shader-subgroup-arithmetic.slang

```diff
@@ -0,0 +1,200 @@
+// not testing glsl or spirv due to crashing issue from some odd code emitting which looks right but fails
```

Comment:
This comment doesn't seem to reflect the current state anymore.
It probably needs to be removed.